mdx/guides/using-the-standard-relayer-api.mdx

```diff
@@ -104,32 +99,24 @@ const [maker, taker] = await web3Wrapper.getAvailableAddressesAsync();
 
 # Declaring decimals and addresses
 
-Since we are dealing with a few contracts, we will specify them now to reduce the syntax load. Fortunately for us, `0x.js` library comes with a couple of contract addresses that can be useful to have at hand. One thing that is important to remember is that there are no decimals in the Ethereum virtual machine (EVM), which means you always need to keep track of how many "decimals" each token possesses. Since we will sell some ZRX for some ETH and since they both have 18 decimals, we can use a shared constant.
+Since we are dealing with a few contracts, we will specify them now to reduce the syntax load. Fortunately for us, `@0x/contract-wrappers` library comes with a couple of contract addresses that can be useful to have at hand. One thing that is important to remember is that there are no decimals in the Ethereum virtual machine (EVM), which means you always need to keep track of how many "decimals" each token possesses. Since we will sell some ZRX for some ETH and since they both have 18 decimals, we can use a shared constant.
 
 ```javascript
 // Token Addresses
-const contractAddresses = getContractAddressesForNetworkOrThrow(
-  NETWORK_CONFIGS.networkId
-);
-const zrxTokenAddress = contractAddresses.zrxToken;
-const etherTokenAddress = contractAddresses.etherToken;
+const zrxTokenAddress = contractWrappers.contractAddresses.zrxToken;
+const etherTokenAddress = contractWrappers.contractAddresses.etherToken;
 ```
 
 0x Protocol uses the ABI encoding scheme for asset data. For example, the ERC20 Token address which is being traded on 0x needs to be encoded. Encoding the address informs the 0x smart contracts on which type of asset is being traded (e.g ERC20 or ERC721) and has optional parameters for different token types (e.g ERC721 token id). In this tutorial we are trading 5 ZRX (ERC20) for 0.1 WETH (ERC20).
 
 ```typescript
-const makerAssetData = assetDataUtils.encodeERC20AssetData(zrxTokenAddress);
-const takerAssetData = assetDataUtils.encodeERC20AssetData(etherTokenAddress);
+const makerAssetData = await contractWrappers.devUtils.encodeERC20AssetData(zrxTokenAddress).callAsync();
+const takerAssetData = await contractWrappers.devUtils.encodeERC20AssetData(etherTokenAddress).callAsync();
```

Comment:
Not necessarily related to the documentation but is there a sync way of doing this? This came up when @xianny  tried removing assetDataUtils from instant.

Reply:
It can be done non-async using AbiEncoder. There are no sync helpers (any more) that I'm aware of, `assetDataUtils` is [soon to be gone](https://github.com/0xProject/0x-monorepo/pull/2373).